### PR TITLE
Re-implement scuba gear for 1.14

### PR DESCRIPTION
**Pull request by tterrag1098**
- Remove dive computer
- Merge into harness
- Make goggles improve visibility underwater

- Remove tanks
- Harness comes with its own tank(s)
- Compressor accepts harness

- Pony bottle now increases air over time instead of instantly

- Remove BCD/Weight Belt

- No recipes yet
- No trimix tier yet

### CODE REVIEW

File: src/main/java/net/tropicraft/core/client/scuba/ModelScubaGear.java
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/client/scuba/ModelScubaGear.java:435]: There's an air hose connecting the backpack to the face mask, I believe that's what this was.

File: src/main/java/net/tropicraft/core/client/scuba/ScubaAmbienceTicker.java
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/client/scuba/ScubaAmbienceTicker.java:41]: nit: we could use item tags here, not a necessary change though.

File: src/main/java/net/tropicraft/core/client/scuba/ScubaHUD.java
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/client/scuba/ScubaHUD.java]: TODO now or later?

File: src/main/java/net/tropicraft/core/common/block/tileentity/AirCompressorTileEntity.java
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/common/block/tileentity/AirCompressorTileEntity.java:154]: so much pi

File: src/main/java/net/tropicraft/core/common/dimension/TropicraftDimension.java
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/common/dimension/TropicraftDimension.java:129]: move to Constants?

File: src/main/java/net/tropicraft/core/client/scuba/ScubaAmbienceTicker.java
Comment (tterrag1098) [reply src/main/java/net/tropicraft/core/client/scuba/ScubaAmbienceTicker.java:41]: I'm not in love with tags that imply hierarchy, users could easily break the code

File: src/main/java/net/tropicraft/core/common/dimension/TropicraftDimension.java
Comment (tterrag1098) [reply src/main/java/net/tropicraft/core/common/dimension/TropicraftDimension.java:129]: I'll do that on the main branch, it's out of scope here. The whole thing needs a refactor to defer to the Dimension always.

File: src/main/java/net/tropicraft/core/client/scuba/ModelScubaGear.java
Comment (tterrag1098) [reply src/main/java/net/tropicraft/core/client/scuba/ModelScubaGear.java:435]: It didn't look right at all. I think it looks okay without it, and debugging the model rotations would take a while.

**Comment by Cojomax99** [comment]
image
Could be nice to show how much air remains in it in the tooltip. A nice-to-have, but a useful nice-to-have

File: src/main/java/net/tropicraft/core/common/item/scuba/ScubaHarnessItem.java
Comment (Cojomax99) [inline src/main/java/net/tropicraft/core/common/item/scuba/ScubaHarnessItem.java:51]: Air shouldn't go down if player is in creative mode

**Comment by Cojomax99** [comment]
Idea for the future: We should totally add a "Fast Flippers" enchantment to go faster while scubaing

**Comment by Cojomax99** [comment]
Missing recipes?

**Review by Cojomax99 — CHANGES_REQUESTED** [review]
I've gameplay tested, and it's super fun! I see a lot of potential with this. My only request is to add recipes. Once it has recipes, it has my approval.

**Review by Cojomax99 — APPROVED** [review]

**Comment by Cojomax99** [comment]
Approving, with the caveat that we need to add recipes for this 🧣

**Pull request merged**
